src/compiler/diagnosticMessages.json

```diff
@@ -659,7 +659,7 @@
         "category": "Error",
         "code": 1207
     },
-    "All files must be modules when the '--isolatedModules' flag is provided.": {
+    "'{0}' cannot be compiled using '--isolatedModules' because it is global. Ensure that it contains an import, export, or an empty 'export {}' statement.": {

```

Comment:
Okay, one last nit:

```suggestion
    "'{0}' cannot be compiled under '--isolatedModules' because it is considered a global script file. Ensure that it contains an import, export, or an empty 'export {}' statement to make it a module.": {
```

Reply:
@DanielRosenwasser Sure. Changed.